app/views/users/index.xlsx.axlsx

```diff
@@ -0,0 +1,69 @@
+wb = xlsx_package.workbook
+col_header = wb.styles.add_style(
+    bg_color: 'FFDFDEDF',
+    b: true,
+    alignment: { horizontal: :center, vertical: :center },
+    border: { color: '00', edges: [:bottom], style: :thin}
+)
+standard_format = wb.styles.add_style alignment: { horizontal: :left }, width: :auto_fit
+date_time_format = wb.styles.add_style format_code: 'dd.mm.yyyy HH:MM'
+date_format = wb.styles.add_style format_code: 'dd.mm.yyyy'
```

Comment:
We already have these in `app/views/assignments/_assignments_table.xlsx.axlsx` and `app/views/clients/_client_table.xlsx.axlsx`, maybe time to extract them into a helper? e.g. a method `excel_styles` which returns a memoized hash of the styles (`@excel_styles ||= { ... }`)

Reply:
I think those styles need to be passed to the actual WorkBook object, so this would only be possible via passing `wb` to the helper.
Using a pratial could be the other option to do it.
But is that realy obvious and explicit?

Reply:
Ah right missed that, it does look a bit ugly if we have to pass the workbook :)

What we could do instead is have a helper to create a new workbook with our default styles, i.e.:

```ruby
module ExcelHelper
  def workbook_with_styles
    wb = xlsx_package.workbook
    wb.styles.add_style[...]
    wb
  end
end

# and then in the view
wb = workbook_with_styles
```

Reply:
that looks convincing to me. I'll do it then.

Reply:
nein, leider fehlanzeige, weil xlsx_package irgend wie nur in views zur verfügung steht:
https://github.com/straydogstudio/axlsx_rails/blob/master/lib/axlsx_rails/template_handler.rb

Reply:
Hmm strange, the helper runs in the same scope as the view so this shouldn't matter. But you're right, it's not really worth the effort ;-)